manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -180,10 +184,12 @@ nmi:
     # - key: "CriticalAddonsOnly"
     #   operator: "Exists"
 
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
   affinity: {}
     # nodeAffinity:
     #   preferredDuringSchedulingIgnoredDuringExecution:
-    #     - preference:
```

Comment:
The indentation needs to be fixed here:

```yaml
      - weight: 1
        preference:
          matchExpressions:
          - key: kubernetes.azure.com/mode
            operator: In
            values:
            - system
```

Reply:
Sorry - copy/paste error! I think 890e7e4cc03560ad0e766df8d22a3ba0153b734f should fix it?